test/cypress/utils/enterSecondPassphrase.js

```diff
@@ -5,6 +5,14 @@ const secondPassphraseSS = {
   secondPassphraseNextBtn: '.second-passphrase-next',
 };
 
+export const enterPassphrase = (passphrase) => {
```

Comment:
This is duplicate to `export function enterSecondPassphraseV2(passphrase) {` down in this file. 
Now, the original `export default function enterSecondPassphrase` is IMO not used anywhere, so `enterSecondPassphraseV2` can be renamed to `enterSecondPassphrase` and replace it